keras_nlp/models/bloom/bloom_decoder.py

```diff
@@ -0,0 +1,218 @@
+# Copyright 2023 The KerasNLP Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# from keras_nlp.backend import keras
+# from keras_nlp.backend import ops
+from keras_nlp.api_export import keras_nlp_export
+from keras_nlp.backend import keras
+from keras_nlp.backend import ops
+from keras_nlp.models.bloom.bloom_attention import BloomAttention
+from keras_nlp.models.bloom.bloom_mlp import BloomMLP
+
+
+@keras_nlp_export("keras_nlp.models.BloomDecoder")
+class BloomDecoder(keras.layers.Layer):
+    def __init__(
+        self,
+        num_heads,
+        dropout=0.0,
+        layer_norm_epsilon=1e-5,
+        kernel_initializer="glorot_uniform",
+        bias_initializer="zeros",
+        **kwargs,
+    ):
+        decoder_sequence_shape = kwargs.pop("decoder_sequence_shape", None)
```

Comment:
I believe this is a weird workaround for multiple argument build methods for Keras 2 only, which we need only for layers that need multiple shapes passed to `build`. Since that is not the case here, we should be able to delete!